### PR TITLE
Contribution guide

### LINKED ISSUE #9: Yaml for custom issues and prs
Add github pr and issue yamls to main branch for a nicer layout and guided questions.

### PR DESCRIPTION
**Pull request by anthonyfabius**
# Pull Request

## Description

I've created the contribution guide. It should have everything anyone should need to contribute to the project. A spell/clarification review is definitely needed because I did not look back as I wrote this. I also removed irrelevant information from the readme.

Closes #9 

## Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] Enhancement (change which alters existing functionality)
- [ ] New feature (change which adds new functionality)
- [ ] This change requires a wiki/documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

Try to checkoff as much as possible if not everything!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+- **Build the Extension:** Then build the chrome extension with:
+
+  ```bash
+  npm run build
+  ```
+
+- **Load the Extension:** Open Google Chrome and go to chrome://extensions/.
+- **Enable Developer Mode:** Toggle on "Developer mode" in the top right corner.
+- **Load Unpacked:** Click on "Load unpacked" and select the EasyApp/extension directory.
+- **Rebuild:** If you make changes to the React Chrome extension or autofill script. Mak
```
Comment (LeadFreeCandy) [inline CONTRIBUTING.md]: typo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+If you're new to Chrome extensions, here are some helpful resources:
+
+- [Official Chrome Extensions Documentation](https://developer.chrome.com/docs/extensions/mv3/getstarted/)
+- [Chrome Extension Development for Beginners](https://www.smashingmagazine.com/2017/04/browser-extension-edge-chrome-firefox-opera-brave-vivaldi/)
+
+If you're new to Typescript, here are some helpful resources:
+
+- [Official Typescript Documentation](https://www.typescriptlang.org/docs/)
+
```
Comment (LeadFreeCandy) [inline CONTRIBUTING.md]: extra newline

File: README.md [hunk 3]
```diff
@@ -23,37 +23,37 @@ Welcome to **EasyApp**, your one-stop solution for navigating job listings with
...
 
 4. Open Chrome and navigate to `chrome://extensions/`.
 5. Enable "Developer mode" (usually a toggle in the top right).
 6. Click on "Load unpacked" and select the `EasyApp/extension` directory.
 7. The **EasyApp** extension icon should now appear in your Chrome toolbar. Click on it and enjoy!
 
-### For Devs
-
-If you alter the react chrome extension or autofill script at all make sure to run `npm run build`. If you alter the extension permissions in the manifest make sure to reload the extension.
-
```
Comment (LeadFreeCandy) [inline README.md:56]: I feel like this should be emphasized somewhere, but this might not the be best place?

**Review by LeadFreeCandy — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (anthonyfabius) [reply README.md:56]: It makes the most sense to put it in the contribution guide imo. Its where we go over setting up the dev build environment and the general workflow of the project. If someone actually wants to contribute to the project they will most likely read the contribution guide. Those not interested in contributing will just read the readme for installation.

File: CONTRIBUTING.md (hunk 2 above)
Comment (anthonyfabius) [reply CONTRIBUTING.md]: The commit 5c4ad46 should resolve this

File: CONTRIBUTING.md (hunk 1 above)
Comment (anthonyfabius) [reply CONTRIBUTING.md]: The commit 5c4ad46 should resolve this.

**Review by LeadFreeCandy — APPROVED** [review]
LGTM!

**Pull request merged**